content/national-qualifications-framework/ncit/content/building-an-online-business/_index.md

```diff
@@ -35,7 +35,7 @@ ncit_standards:
 - 115385
 prerequisites:
   hard:
-  - projects/first-personal-website
+  - projects/web-development-website
```

Comment:
This should have the simple website as a prerequisite. Otherwise we are forcing the data scientists to do web dev on hard mode. We are trying to avoid that